specs/lib/loadMtlSpec.js

```diff
@@ -1,43 +1,459 @@
+            var pbr = material.pbrMetallicRoughness;
+            expect(pbr.baseColorTexture).toBeUndefined();
+            expect(pbr.metallicRoughnessTexture).toBeUndefined();
+            expect(pbr.baseColorFactor).toEqual([0.5, 0.5, 0.5, 1.0]);
+            expect(pbr.metallicFactor).toBe(0.0); // No metallic
+            expect(pbr.roughnessFactor).toBe(1.0); // Fully rough
+            expect(material.emissiveTexture).toBeUndefined();
+            expect(material.normalTexture).toBeUndefined();
+            expect(material.ambientTexture).toBeUndefined();
+            expect(material.emissiveFactor).toEqual([0.0, 0.0, 0.0]);
+            expect(material.alphaMode).toBe('OPAQUE');
+            expect(material.doubleSided).toBe(false);
+        });
+
+        it('creates material with textures', function() {
+            options.metallicRoughness = true;
+
+            var material = loadMtl._createMaterial({
+                diffuseTexture : diffuseTexture,
+                ambientTexture : ambientTexture,
+                normalTexture : normalTexture,
+                emissiveTexture : emissiveTexture,
+                specularTexture : specularTexture,
+                specularShininessTexture : specularShininessTexture
+            }, options);
+
+            var pbr = material.pbrMetallicRoughness;
+            expect(pbr.baseColorTexture).toBeDefined();
+            expect(pbr.metallicRoughnessTexture).toBeDefined();
+            expect(pbr.baseColorFactor).toEqual([1.0, 1.0, 1.0, 1.0]);
+            expect(pbr.metallicFactor).toBe(1.0);
+            expect(pbr.roughnessFactor).toBe(1.0);
+            expect(material.emissiveTexture).toBeDefined();
+            expect(material.normalTexture).toBeDefined();
+            expect(material.occlusionTexture).toBeDefined();
+            expect(material.emissiveFactor).toEqual([1.0, 1.0, 1.0]);
+            expect(material.alphaMode).toBe('OPAQUE');
+            expect(material.doubleSided).toBe(false);
+        });
+
+        it('packs occlusion in metallic roughness texture', function() {
+            options.metallicRoughness = true;
+            options.packOcclusion = true;
+
+            var material = loadMtl._createMaterial({
+                ambientTexture : alphaTexture,
+                specularTexture : specularTexture,
+                specularShininessTexture : specularShininessTexture
+            }, options);
+
+            var pbr = material.pbrMetallicRoughness;
+            expect(pbr.metallicRoughnessTexture).toBeDefined();
+            expect(pbr.metallicRoughnessTexture).toBe(material.occlusionTexture);
+        });
+
+        it('does not create metallic roughness texture if decoded texture data is not available', function() {
+            options.metallicRoughness = true;
+            options.packOcclusion = true;
+
+            var material = loadMtl._createMaterial({
+                ambientTexture : ambientTexture, // Is a .gif which can't be decoded
```

Comment:
The warning is provided later, but only if the PBR texture is created. Otherwise it will just ignore that it couldn't be decoded.

https://github.com/AnalyticalGraphicsInc/obj2gltf/pull/67/files#diff-959e13a8da36b7c10b96349c54350791R323

https://github.com/AnalyticalGraphicsInc/obj2gltf/pull/67/files#diff-959e13a8da36b7c10b96349c54350791R199